pkg/sql/colexecop/operator.go

```diff
@@ -91,7 +91,12 @@ type KVReader interface {
 	// GetKVCPUTime returns the CPU time consumed *on the current goroutine* by
 	// KV requests. It must be safe for concurrent use. It is used to calculate
 	// the SQL CPU time.
+	// TODO(alyshan): think of naming.
```

Comment:
GetKVCPUTime currently is used so that operators (which serve vectorized queries?) can calculate an accurate "SQL CPU Time" which is currently recorded with SQL Stats.

If the KV CPU Time approach in this PR is acceptable, and thus we can collect it for every query execution - we should aim to improve SQL CPU time tracking by:
- collecting Sql Cpu Time on every execution instead of only when tracing is enabled
- collecting Sql Cpu Time for non vectorized queries as well.